watertap/data/techno_economic/surface_discharge.yaml

```diff
@@ -1,9 +1,49 @@
 default:
-  recovery_frac_mass_H2O:
-    value: 0.9999
-    units: dimensionless
-    reference: Lenntech https://www.lenntech.com/processes/surface-water-discharge-of-brine.htm.  Electricity
-      consumption is based on Carlsbad data.
-  default_removal_frac_mass_solute:
-    value: 0
-    units: dimensionless
+  capital_cost:
+    cost_factor: None
+    reference_state:
+      value: 10417
+      units: m^3/hr
+    capital_a_parameter:
+      value: 35e6 # TODO does this need to be scaled to account for $ vs MM$ ?
+      units: USD_2020
+    capital_b_parameter:
+      value: 0.873
+      units: dimensionless
+    pipe_cost_basis:
+      value: 35000
+      units: USD_2020/(miles*inches)
+  lift_height:
+    value: 100
+    units: ft
+  pipe_diameter:
+    value: 8
+    units: inches
+  pipe_distance:
+    value: 0 # default case includes no pipe costs
+    units: miles
+emwd:
+  capital_cost:
+    basis: flow_vol
+    cost_factor: None
+    reference_state:
+      value: 10417
+      units: m^3/hr
+    capital_a_parameter:
+      value: 35e6 # TODO does this need to be scaled to account for $ vs MM$ ?
```

Comment:
Most $ values in the WT3 code were for millions of dollars ,so we generally have to multiply by 1e6 in our YAML files (alternatively, you could set the cost units as `MUSD_2020`, but we have generally just used 1e6).

Reply:
No, I don't think so. I just wanted to clarify,